assets/javascripts/modules/responsiveInlineDetails.js

```diff
@@ -0,0 +1,30 @@
+module.exports = function() {
+
+  $('[data-responsive-inline-details]').each(function() {
+    var $detailsHolder,
+    $detailsTargets,
+    $detailsButton;
+
+    $detailsHolder = $(this);
+    $detailsButton = $detailsHolder.find('.inline-details-toggle');
+    $detailsTargets = $('.' + $detailsHolder.data('responsive-inline-details'));
+
+    $detailsButton.on('click', function(e) {
```

Comment:
No reason, you're quite right I believe when I was scoping this out I had some additional code in the click listener that's not there anymore. I'll use the following approach:

$detailsButton.on('click', detailsButtonClick)